static/app/components/events/interfaces/exceptionStacktraceContent.tsx

```diff
@@ -61,6 +67,27 @@ const ExceptionStacktraceContent = ({
    * It is easier to fix the UI logic to show a non-empty stack trace for chained exceptions
    */
 
+  const stackTracePlatform =
+    ((data?.frames ?? []).find(frame => frame.platform) || platform) ?? 'other';
+
+  if (
+    hasGroupingTreeUI &&
+    (stackTracePlatform === 'native' || stackTracePlatform === 'cocoa')
```

Comment:
I'm wondering if it would be better to show StacktraceContentV2 for all platforms even if the other platforms haven't been implemented yet. After all one has the labels and the other one doesn't.

Reply:
I think I would prefer to leave it as it is until I check what the UI looks like for other platforms